### ISSUE TITLE
Missing run_file functionality

### ISSUE THREAD

**Issue by MxmUrw** [Marwes/gluon#127]
For running gluon code there are 3 methods available:
- `load_script`
- `load_file`
- `run_expr`

I would like to run a file and access its result inside rust, i.e. I would like to have a "`run_file`" method. Currently there seems to be no easy way to do this.

As a quick fix I copied most of the `load_file` code, calling into `run_expr` instead of `load_script`, is this the right approach?

**Comment by Marwes** [comment]
Yep, that should work fine, running from a file instead of a string is just a matter of reading the file into a string and running that string.

**Comment by brendanzab** [comment]
It might pay to look at the functions we have and decide what the best API is, given that Rust already has a file system API.

**Comment by MxmUrw** [comment]
Well, you are probably right, this is not really an issue. Using `run_expr` should be enough.

**Comment by brendanzab** [comment]
I think it is more that we might have a confusing API - when you supply one thing, it is reasonable to wonder why the others have not been implemented. So you were right to bring it up!

**Comment by Marwes** [comment]
The `gluon` library itself is more or less a bunch of re-exports and convenience functions so having functions for running program both from file and string would not be out of place.
